finance/api/migrations/0001-create-payments.js

```diff
@@ -0,0 +1,49 @@
+'use-strict'
```

Comment:
Sobre a nomenclatura incluindo números no início do arquivo, não é um erro, mas pessoalmente não vejo muito esta estratégia sendo utilizada.